database/models/soundtracks.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+const soundSchema = new mongoose.Schema({
+  bundle_id: { type: Number, unique: true },
+  bonus_info: [
+    {
+      title: String,
+      composer_performer: String,
+      cover: String,
+      tracklist: [
+        {
+          track_number: String,
+          name: String,
+          song: String
+        }
+      ]
+    }
+  ]
+});
+
+const Soundtrack = mongoose.model('Soundtrack', soundSchema);
+
+module.exports = Soundtrack;
```

Comment:
I believe the airbnb style guide requires a new line at the end of files. Two of the files in this PR have a new line and two don't. Either way, you should pick one and stick with it.

Reply:
Committing to a new line at the end of the file always. Nice catch.